pkg/util/ovs/parse.go

```diff
@@ -361,3 +361,35 @@ func fieldMatches(val, match string, ptype ParseType) bool {
 
 	return false
 }
+
+func ParseExternalIDs(externalIDs string) (map[string]string, error) {
+	ids := make(map[string]string, 1)
+	// Output from "find" and "list" will have braces, but input to "set" won't
+	if externalIDs[0] == '{' && externalIDs[len(externalIDs)-1] == '}' {
+		externalIDs = externalIDs[1:len(externalIDs)-1]
+	}
+	for _, id := range strings.Split(externalIDs, ",") {
```

Comment:
would strings.Fields or strings.FieldsFunc make it better?
Split/Trim combos are delicate.

Reply:
I kinda want unit tests for the Parse/Unparse functions too.

Reply:
Yes, considering this is common for find, list and set ops, unit tests will be helpful.